### ISSUE TITLE
about proxy-nca loss

### ISSUE THREAD

**Issue by YanhuaCheng** [BG2CRW/top_k_optimization#1]
loss = torch.sum(- T * F.log_softmax(D, -1), -1), should D be -D since we want to minimize the distance between the example with its proxy with same label.
